src/NServiceBus.Core/Recoverability/Faults/FaultToDispatchConnector.cs

```diff
@@ -25,7 +24,7 @@ public override Task Invoke(IFaultContext context, Func<IRoutingContext, Task> s
                 }
             }
 
-            var dispatchContext = new RoutingContext(message, new UnicastRoutingStrategy(context.ErrorQueueAddress), context);
+            var dispatchContext = this.CreateRoutingContext(context);
```

Comment:
Bit on the fence what is better. The beauty of having the unicast strategy here is that you see where we route to. Hiding everything in the extension method is obscuring that a little bit.